components/brave_ads/common/notification_ad_feature.cc

```diff
@@ -11,8 +11,17 @@ BASE_FEATURE(kNotificationAdFeature,
              "NotificationAds",
              base::FEATURE_ENABLED_BY_DEFAULT);
 
+BASE_FEATURE(kAllowedToFallbackToCustomNotificationAdFeature,
+             "AllowedToFallbackToCustomAdNotifications",
```

Comment:
`AllowedToFallbackToCustomNotificationAds`, would be great to check elsewhere so that we use `NotificationAd` and preferably not `AdNotification`. Thanks